### ISSUE TITLE
Is there an option to use setSearchTerm without input?

### ISSUE THREAD

**Issue by NiedbalskiPatryk** [elastic/search-ui#531]
Hi.

At the beginning I would like to thank you for the cool package you have created.

I would like to use two AppSearch instances in my application, one that handles the list of products, and the other that searches for related products on click event (they will be presented in the modal). Second instance will not trackUrlState.

The problem is that when I try to execute the ctx.setSearchTerm ('some-slug') function in the middle of WithSearch, it sets a string to the ctx.searchTerm position but does not run the search.

Can I somehow run setSearchTerm without input?

Regards PN

setSearchTerm not working on : 
![image](https://user-images.githubusercontent.com/60225407/102992315-4a6ca600-451b-11eb-8120-ee2f3093aa55.png)

setSearchTerm work on :
![image](https://user-images.githubusercontent.com/60225407/102992391-6c662880-451b-11eb-9080-29e53c603fbf.png)

**Comment by JasonStoltz** [comment]
Could you rephrase your question? I am not sure I understand. Any time you call the `setSearchTerm` action, it should trigger a query.
